### PR TITLE
added function to summrize text in the killring-0.

### PR DESCRIPTION
**Pull request by alarawms**
I made a new function to summrize text in killring from any source. web or any app. 

tested on mac. and working. 
Screenshot 2024-04-10 at 4 10 27 AM
Screenshot 2024-04-10 at 4 11 19 AM

### CODE REVIEW

**Comment by s-kostyaev** [comment]
Hi @alarawms 
Please remove unnecessary empty lines from your PR

**Comment by alarawms** [comment]
done

File: ellama.el [hunk 1]
```diff
@@ -1262,6 +1263,16 @@ ARGS contains keys for fine control.
...
+;;;###autoload
+(defun ellama-summarize-killring ()
+  "Summarize text from the kill ring."
+  (interactive)
+  (let ((text (current-kill 0)))
+    (if (string-empty-p text)
+        (message "No text in the kill ring to summarize.")
+      (ellama-instant (format ellama-summarize-prompt-template text)))))
+
+
```
Comment (s-kostyaev) [inline ellama.el:1275]:
```suggestion
```

**Comment by s-kostyaev** [comment]
@alarawms Thank you for contribution

**Pull request merged**